src/docker/run-apache.sh

```diff
@@ -51,8 +51,6 @@ LOCATION_DIRECTIVES=$(cat << EOF
   </Location>
 
   <Location "/service/api">
```

Comment:
I think you could remove this entire Location block - the one above it is a superset.